packages/web/e2e/pages/swap-page.ts

```diff
@@ -0,0 +1,161 @@
+import { expect, Locator, Page } from "@playwright/test";
+
+export class SwapPage {
+  readonly page: Page;
+  readonly connectWalletBtn: Locator;
+  readonly swapBtn: Locator;
+  readonly swapHalfBtn: Locator;
+  readonly swapMaxBtn: Locator;
+  readonly swapInput: Locator;
+  readonly kepltWalletBtn: Locator;
+  readonly flipAssetsBtn: Locator;
+  readonly exchangeRate: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.connectWalletBtn = page
+      .getByRole("button", { name: "Connect wallet", exact: true })
+      .first();
+    this.swapBtn = page.getByRole("button", { name: "Swap", exact: true });
+    this.swapHalfBtn = page.getByRole("button", { name: "HALF", exact: true });
+    this.swapMaxBtn = page.getByRole("button", { name: "MAX", exact: true });
+    this.swapInput = page.locator('input[type="number"]');
+    this.kepltWalletBtn = page.locator("button").filter({ hasText: /^Keplr$/ });
+    this.flipAssetsBtn = page.locator(
+      '//div/button[contains(@class, "ease-bounce")]'
+    );
+    this.exchangeRate = page.locator('//span[contains(@class, "subtitle2")]');
+  }
+
+  async goto() {
+    const assetPromise = this.page.waitForRequest("**/assets.json");
+    await this.page.goto("/");
+    const request = await assetPromise;
+    console.log(request.url());
+    // we expect that after 2 seconds tokens are loaded and any failure after this point should be considered a bug.
+    await this.page.waitForTimeout(2000);
+  }
+
+  async connectWallet(promise: Promise<Page>) {
+    await this.connectWalletBtn.click();
+    await this.kepltWalletBtn.click();
+    // Handle Pop-up page ->
+    const newPage = await promise;
+    await newPage.waitForLoadState();
+    console.log(newPage.url());
```

Comment:
Is this console log intended?

Reply:
yes, but it is for a debugging only. I will remove it for now.